lib/ws_servers/api/send_recurring_algo_order_list.js

```diff
@@ -6,17 +6,36 @@ const sendError = require('../../util/ws/send_error')
 const mapRecurringAlgoOrderState = require('../../util/map_recurring_ao_state')
 const restPaginationHandler = require('../../util/rest_pagination_handler')
 
+const getRecurringAosWithRetry = async (rest, debug) => {
+  let algoOrders = []
+  let retries = 0
+  let receivedData = false
+  let err = null
+  while (!receivedData && retries < 3) {
+    try {
+      algoOrders = await restPaginationHandler(rest.getRecurringAlgoOrders.bind(rest), {})
+      receivedData = true
+      err = null
+      debug('fetched total Recurring AOs', algoOrders.length)
```

Comment:
return `algoOrders` after debug so we can avoid a whole logic check.

Reply:
Fixed.